<?xml version="1.0" encoding="utf-8"?>
<launch>

  <node name="urg_node" pkg="urg_node" type="urg_node" output="screen">
    <param name="ip_address"         value=""/>
    <param name="serial_port"        value="/dev/ttyACM0"/>
    <param name="serial_baud"        value="115200"/>
    <param name="frame_id"           value="base_laser_link"/>
    <param name="calibrate_time"     value="false"/>
    <param name="publish_intensity"  value="true"/>
    <param name="publish_multiecho"  value="false"/>
    <param name="angle_min"          value="-1.5707963"/>
    <param name="angle_max"          value="1.5707963"/>
  </node>

  <node name="laser_to_image" pkg="pd3_demo" type="laser_to_image.py" output="screen">
    <param name="disc_size" type="double" value="0.01" />
    <param name="max_range" type="double" value="2.5" />
  </node>
</launch>
